egs/swbd/s5c/local/chain/run_lstm_d.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+# based on run_tdnn_2o.sh
+
+set -e
+
+# configs for 'chain'
+stage=10
+train_stage=-10
+get_egs_stage=-10
+speed_perturb=true
+dir=exp/chain/lstm_d  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+decode_dir_affix=
+
+# LSTM options
+splice_indexes="-2,-1,0,1,2 0 0"
+lstm_delay=" -3 -3 -3 "
+label_delay=5
+num_lstm_layers=3
+cell_dim=1024
+hidden_dim=1024
+recurrent_projection_dim=256
+non_recurrent_projection_dim=256
+include_log_softmax=true
```

Comment:
Oops thats a bug. Corrected it.

On Sat, Feb 27, 2016 at 5:09 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/chain/run_lstm_d.sh
> <https://github.com/kaldi-asr/kaldi/pull/525#discussion_r54340378>:
>
> > +get_egs_stage=-10
> > +speed_perturb=true
> > +dir=exp/chain/lstm_d  # Note: _sp will get added to this if $speed_perturb == true.
> > +decode_iter=
> > +decode_dir_affix=
> > +
> > +# LSTM options
> > +splice_indexes="-2,-1,0,1,2 0 0"
> > +lstm_delay=" -3 -3 -3 "
> > +label_delay=5
> > +num_lstm_layers=3
> > +cell_dim=1024
> > +hidden_dim=1024
> > +recurrent_projection_dim=256
> > +non_recurrent_projection_dim=256
> > +include_log_softmax=true
>
> not sure why this is set to true- would be better IMO to hardcode it to
> false rather than having a variable. chain models do not need the final
> log-softmax and actually it's incompatible with l2-regularize
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/525/files#r54340378>.
>